src/test_l1_dsVideoPort.c

```diff
@@ -91,6 +92,21 @@ static int gTestID = 1;
 	}\
 }\
 
+/* Global flags to support features */
+static bool extendedEnumsSupported=false; //Default to not supported
+
+#define CHECK_FOR_EXTENDED_ERROR_CODE( result, enhanced, old )\
```

Comment:
#defines have GLOBAL Scope, you'll be re-using the same macro in all files..

therefore has to be module specific

`DS_VIDEO_PORT_CHECK_FOR_EXTEND_ERROR_CODE`

it's only working by luck.